common/scripts/mesh-default.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
This is setting default gateway.  Should it be renamed to mesh-default-gw.sh

Reply:
Agreed